src/api/math/float/fmae.rs

```diff
@@ -5,21 +5,9 @@ macro_rules! impl_math_float_fmae {
         impl $id {
             /// Fused multiply add estimate: ~= `self * y + z`
             ///
-            /// On some architectures, it is possible to combine a multiply
-            /// followed by an addition in a single instruction.
-            /// Besides performance, this may also offer better precision
-            /// than performing the operations individually.
-            ///
-            /// Note that using this function does **not** guarantee that a FMA
-            /// instruction will be emitted;
-            /// the architecture may not support it, or the compiler may decide
-            /// it's more efficient in a specific case not to use it.
-            ///
-            /// Most architectures which have support for FMA
-            /// also have an equivalent version of this function,
-            /// fused multiply subtract (`self * y - z`).
-            /// Simply negating the second parameter of this function
-            /// will make the compiler generate it.
+            /// While fused-multiply add ([fma]) has infinite precision,
```

Comment:
I think you need to put fma in ticks `\`fma\`` for the doc link to work properly.